### ISSUE TITLE
[FEATURE REQUEST] Multiple Policies for or

### ISSUE THREAD

**Issue by TheFinestArtist** [balderdashy/sails#1571]
This is current multiple policies
```
UserController: {
    lock: ['isLoggedIn', 'isAdmin']
}
```

It will be really great if there is logic for multiple policies such as (&& || !)
```
UserController: {
    find: [ ['isDevelopment' || 'isStaging' || 'isAdmin'] && !'isUser']
}
```
I don't know think my example is quite nice though.

**Comment by mikermcneil** [comment]
Thanks for posting, @TheFinestArtist.  I'm an experimental issue cleanup bot-- nice to meet you!

It has been a couple of months since there have been any updates or new comments on this page.  If this issue has been resolved, please feel free to disregard the rest of this message.On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, etc.  Please include a link back to this page for reference.
  + add a comment to _this_ issue with a link to the new issue (for people arriving from search engines)

Thanks so much for your help!

<3
Sails.js proto-bot
